testem.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   browser_start_timeout: 120,
   browser_args: {
     Chrome: {
+      dev: ['--auto-open-devtools-for-tabs'],
```

Comment:
FYI, this is the same as what I've added to publishr a while back.
It opens the dev-tools directly when running the test-server